extensions/database/tests/conf/github_actions_tests.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suite name="DatabaseExtensionUnitTestSuite">
+    <parameter name = "mySqlDbName" value="test_db"/> 
+    <parameter name = "mySqlDbHost" value="127.0.0.1"/> 
+    <parameter name = "mySqlDbPort" value="3306"/> 
+    <parameter name = "mySqlDbUser" value="root"/>
+    <parameter name = "mySqlDbPassword" value="root"/>
+    <parameter name = "mySqlTestTable" value="test_table"/>
+    
+    <parameter name = "pgSqlDbName" value="test_db"/> 
+    <parameter name = "pgSqlDbHost" value="postgres"/> 
+    <parameter name = "pgSqlDbPort" value="5432"/> 
+    <parameter name = "pgSqlDbUser" value="postgres"/>
+    <parameter name = "pgSqlDbPassword" value="postgres"/>
+    <parameter name = "pgSqlTestTable" value="test_table"/>
+    
+    <parameter name = "mariaDbName" value="test_db"/> 
+    <parameter name = "mariaDbHost" value="127.0.0.1"/> 
+    <parameter name = "mariadbDbPort" value="3306"/> 
+    <parameter name = "mariaDbUser" value="root"/>
+    <parameter name = "mariaDbPassword" value="root"/>
+    <parameter name = "mariaDbTestTable" value="test_table"/>
```

Comment:
I don't see this database being started anywhere.

Reply:
Yes indeed, I realized that so far in Travis we actually tested the MariaDB connector with the MySQL instance (since they both connected to port 3306). It could be worth spinning a real MariaDB instance instead.